README.adoc

```diff
@@ -323,41 +309,46 @@ include::finish/src/test/java/it/io/openliberty/guides/system/SystemEndpointTest
 // =================================================================================================
 
 [role='command']
-include::{common-includes}/mvnverify.adoc[]
+include::{common-includes}/devmode-test.adoc[]
+
+You will see the following output:
```

Comment:
Change
"You will see the following output:"
To
"You see the following output:"

Reply:
Updated